lib/push.js

```diff
@@ -195,7 +195,14 @@ module.exports = function (log, db, config) {
       canSendToIOSVersion = deviceVersion => deviceVersion >= 10.0
       break
     default:
-      canSendToIOSVersion = () => false
+
+      if (command) {
```

Comment:
Rather than having an `if` inside a `switch`, it might be slightly cleaner to coerce `command` to `false` or `null` or something above, and then match explicitly on it as a case in the switch.  For example:

```
let command = (payload && payload.command) || null
switch (command) {
  ...
  case null:
    // new logic here
  default:
    // existing logic here
}
```